src/lightning_app/cli/pl-app-template/app.py

```diff
@@ -4,10 +4,10 @@
 from core.components import TensorBoard, WeightsAndBiases
 from core.components.script_runner import ScriptRunner
 
-from lightning.app import LightningApp, LightningFlow
-from lightning.app.frontend import StaticWebFrontend
-from lightning.app.storage import Path
-from lightning.app.utilities.packaging.cloud_compute import CloudCompute
+from lightning_app import LightningApp, LightningFlow
```

Comment:
Also, this will be replaced back when you'll be using `lightning` package, coming in #15105